### ISSUE TITLE
Logging raw logit_scale before .exp() operation

### ISSUE THREAD

**Issue by ChenDelong1999** [mlfoundations/open_clip#104]
In the previous version, `logit_scale` is clamped and logged by directly accessing `model.logit_scale`:
```python
# Initializing
self.logit_scale = nn.Parameter(torch.ones([]) * np.log(1 / 0.07)) # self.logit_scale = 2.659
# Clamping
model.logit_scale.data = torch.clamp(model.logit_scale.data, 0, 4.6052) # ln(100) = 4.6052
# Logging
logging.info(f"logit_scale {model.logit_scale.data:.3f}")
```
So the initial logit_scale in the log is displayed as 2.659.

Current version logs the  `logit_scale` value returned by `model.forward()`, which is after the `.exp()` operation:
```python
def forward(self, image, text):     
    ...
    return image_features, text_features, self.logit_scale.exp()
```
witch makes the logit_sclae in the log to be 14.286. It is kind of confusing since the clamping is still done by accessing `model.logit_scale`, i.e. clamping 2.659 between 0 and 4.6052.

😀 I kindly suggest to log logit_scale by directly accessing `model.logit_scale`, so it is possible to watch how close is the current logit_scale to the maximum allowed value i.e. ln(100)=4.6052.

**Comment by ChenDelong1999** [comment]
Or simply change [this line](https://github.com/mlfoundations/open_clip/blob/main/src/training/train.py#L114) from:
```python
logit_scale_scalar = logit_scale.item()
```
to
```python
logit_scale_scalar = np.log(logit_scale.item())
```

**Comment by rwightman** [comment]
@ChenDelong1999 I don't see how it makes a difference other than being more intuitive when it's viewed 1..100 v 0..4.6052. Also the scale.exp() is what the logits are being scaled by so again, makes more sense to show that. I don't plan to change it back but as you note it's easy to do so... and you know the maximum allowed is 100....

**Comment by ChenDelong1999** [comment]
OK, thanks.
